simpeg/potential_fields/magnetics/simulation.py

```diff
@@ -1665,562 +1664,531 @@ def _gtg_diagonal_without_building_g(self, weights):
 
 
 class Simulation3DDifferential(BaseMagneticPDESimulation):
-    """
-    Secondary field approach using differential equations!
+    r"""A secondary field simulation for magnetic data.
+
+    Parameters
+    ----------
+    mesh : discretize.base.BaseMesh
+    survey : magnetics.survey.Survey
+    mu : float, array_like
+        Magnetic Permeability Model (H/ m). Set this for forward
+        modeling or to fix while inverting for remanence. This is used if
+        muMap == None
+    muMap : SimPEG.maps.IdentityMap, optional
+        The mapping used to go from the simulation model to `mu`. Set this
+        to invert for `mu`.
+    rem : float, array_like
+        Magnetic Polarization \mu_0*M (nT). Set this for forward
```

Comment:
Similar comment here, `\mu_0` could be baked in the computation, and user could invert for an "effective susceptibility" where M = H_0 * k_effective. This would bring both quantities to invert for be dimensionally equivalent.